lib/parse.js

```diff
@@ -53,7 +53,7 @@
 
     let sgn = Math.sign(firstFloat);
     if (sgn === 0) {
-      sgn = 1;
+       sgn = strIn.startsWith('-') ? -1 : 1;
```

Comment:
Should use `str` instead, as it has the white-space already trimmed. But still better than before. Pulling as-is, will tweak later. Thanks!